src/components/Swap.tsx

```diff
@@ -322,6 +327,22 @@ const Swap = ({ transferChains }: SwapProps) => {
   // Elements used for animations
   const routeCards = useRef<HTMLDivElement | null>(null)
 
+  useEffect(() => {
+    // get new execution status on page load
+    activeRoutes.map((route) => {
+      if (!web3 || !web3.library) return
+      const settings = {
+        updateCallback: (updatedRoute: RouteType) => {
+          storeRoute(updatedRoute)
+          setActiveRoutes(readActiveRoutes())
+          setHistoricalRoutes(readHistoricalRoutes())
+        },
+      }
```

Comment:
I guess the "old" update callback that we passed when we initially started the route doesn't exist anymore after a page load and that's why we need to pass a new one?

Reply:
Yes exactly. The SDK is not saving anything anywhere and starts blank!
In general we should not access browser specific stuff (storage and other apis) in the sdk as it could be used on servers and cmd-lines as well